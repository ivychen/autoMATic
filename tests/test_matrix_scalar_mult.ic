int main() {
    int matrix[2][2] a = [[1, 2]; [0, 1]] * 42;
    
    int i = 0, j = 0;
    for (i = 0; i < 2; i++) {
        for (j = 0; j < 2; j++) {
            println(a[i][j]);
        }   
    }
}
