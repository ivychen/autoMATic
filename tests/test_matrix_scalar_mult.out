42
84
0
42
